app/models/casa_case.rb

```diff
@@ -200,7 +200,7 @@ def set_slug
   end
 
   def full_attributes_hash
-    attributes.symbolize_keys.merge({contact_types: casa_case_contact_types.map(&:attributes), court_orders: case_court_orders.map(&:attributes)})
+    attributes.symbolize_keys.merge({contact_types: contact_types.reload.map(&:attributes), court_orders: case_court_orders.map(&:attributes)})
```

Comment:
DB Updates happen in L178-181 however by reloading the contact_types, we get the updated contact_types.